VEX/priv/host_riscv64_defs.h

```diff
@@ -455,6 +456,14 @@ typedef struct {
          HReg            base;
          Int             soff12; /* -2048 .. +2047 */
       } FpLdSt;
+      /* Floating-point conditional-select pseudoinstruction. */
+      struct {
+         IRType ty;
```

Comment:
Backend instruction definitions should preferably not use any "midend" IR types. That slightly violates the layering.

I would actually remove this type information altogether. It is not needed as using FMV.D should be fine for both `Ity_F32` and `Ity_F64`.